locales/nl/LC_MESSAGES/volto.po

```diff
@@ -1276,6 +1276,10 @@ msgstr "Sitemap"
 msgid "Site Setup"
 msgstr "<<<<<<< HEAD"
```

Comment:
Uh. Unrelated to this PR. Seems there is a merge leftover in the NL translations.